slurm/slurm.sh

```diff
@@ -217,16 +221,16 @@ slurm_monitor_job() {
     local returnStdout=${6:-'no'}
     local quiet=${7:-'no'}
 
-
     warn "Monitor style: $monitorStyle" "$quiet"
 
     # Delete any prior logs
-    if [ -n "$jobStdout" ]; then
-        jobStderr=$(echo -e "$jobStdout" | sed s/.out$/.err/)
-        rm -rf $jobStdout $jobStderr
-    fi
-    # If a log file is provided and viewLogOutput is 'yes', then start tailing the files
+    # if [ -n "$jobStdout" ]; then
+    #     warn "here if $jobStdout"
+          jobStderr=$(echo -e "$jobStdout" | sed s/.out$/.err/)
+    #     rm -rf $jobStdout $jobStderr
+    # fi
```

Comment:
This was removing SLURM generated output files... but we need similar mechanism in the beginning of the script to remove existing log/err file